### ISSUE TITLE
Avoid using flex-direction as column-reverse or row-reverse

### ISSUE THREAD

**Issue by wulforr** [jackdomleo7/Checka11y.css#55]
### Describe the new a11y feature or project enhancement
Screen Readers read the data according to the dom nodes so if the flex-direction is set to column-reverse or row-reverse then screen readers wont be able to read them correctly.

### Describe the solution you'd like
I cant think of a solution for this more discussion is required.

### Link(s)

[https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction#Accessibility_concerns](
https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction#Accessibility_concerns)

**Comment by jackdomleo7** [comment]
I like the idea of this, sounds like this should be shown as a "WARNING", though. 🙂

I'm curious, how can we use CSS to test for CSS, especially because CSS doesn't affect the HTML structure. 🤔

**Comment by danspratling** [comment]
There are also perfectly valid reasons to have the content change direction.

One example I use regularly is a left/right content grid like so, where the content would naturally stack (image > text > image > text) but to make it more visually appealing on desktop some of the rows are reversed (with the content remaining in the same order) which doesn't change the context.

![Screenshot 2020-10-19 at 17 56 00](https://user-images.githubusercontent.com/11349104/96485581-87bf6980-1234-11eb-8969-93b1d9b0aacf.png)

So we'd not only need to check if the content was changing direction but also if the context of the content changes if it direction, which is much harder to do (and probably impossible in css)

**Comment by jackdomleo7** [comment]
Actually, you do make excellent points there @danspratling. I do this all the time without actually realising. 🤔

Closing as if we wanted to do this, we can't because it's impossible with just CSS.
